Proofreading edits

josephstein opened the pull request.

josephstein committed:
proofreading

Rokt33r commented on packages/mapped-reducer/readme.md:
```diff
@@ -125,7 +125,7 @@ Get a subreducer for the given action type(s).
 #### `#reduce(state: State, action: Action): State`
 
-This is the actual reducing method. You should pass this to `createStore` or `combineReducers`.
+This is the actual reducing method. You should pass this to `createStore` or `combinedReducers`.
```
`combineReducers ` is not a mistake. It is name of a function from Redux.
I'll fix this.

josephstein replied:
ah ok. I thought you were referencing a variable from the code snippet

Rokt33r committed:
Merge branch 'master' into readme-editing

Rokt33r committed:
Revert a change, `combinedReducers`

Rokt33r committed:
Add joseph to authors

Rokt33r committed:
Fix wrong code example

Rokt33r committed:
Publish

 - typed-redux-kit.base@1.0.2
 - typed-redux-kit.batch-enhancer@1.0.2
 - typed-redux-kit.mapped-reducer@1.0.2
 - typed-redux-kit.trackable@1.0.2
 - typed-redux-kit@1.0.2
